### PR TITLE
Make screenshot tolerance stricter by default

### PR DESCRIPTION
**Pull request by yschimke**
#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

File: sample/src/test/snapshots/images/com.google.android.horologist.screensizes_MediaPlayerTest_screenshot[0]_mobvoiticwatchpro5.png
Comment (yschimke) [inline sample/src/test/snapshots/images/com.google.android.horologist.screensizes_MediaPlayerTest_screenshot[0]_mobvoiticwatchpro5.png:1]: I need to dig in, whether this is RNG native mode, or a real bug.

File: media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui.components_LoadingMediaDisplayTest_loadingMediaDisplay_textMediaDisplay_overlay.png
Comment (yschimke) [inline media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui.components_LoadingMediaDisplayTest_loadingMediaDisplay_textMediaDisplay_overlay.png:1]: This looks like a bug we missed...

File: compose-material/src/test/snapshots/images/com.google.android.horologist.compose.material_ChipTest_withLongText.png
Comment (yschimke) [inline compose-material/src/test/snapshots/images/com.google.android.horologist.compose.material_ChipTest_withLongText.png:1]: Will confirm if this is RNG or real...

File: images/coil/src/test/snapshots/images/com.google.android.horologist.compose.material_ChipTest_withPlaceholderIcon.png
Comment (yschimke) [inline images/coil/src/test/snapshots/images/com.google.android.horologist.compose.material_ChipTest_withPlaceholderIcon.png:1]: Will confirm if resource loading or RNG....

File: sample/src/test/snapshots/images/com.google.android.horologist.screensizes_MediaPlayerTest_screenshot[0]_mobvoiticwatchpro5.png
Comment (yschimke) [reply sample/src/test/snapshots/images/com.google.android.horologist.screensizes_MediaPlayerTest_screenshot[0]_mobvoiticwatchpro5.png:1]: RNG bug.  Will file once RNG hardware mode is stable.

File: media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui_MediaPlayerStatesScreenTest_mediaPlayerScreen[1]_notconnected.png
Comment (luizgrp) [inline media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui_MediaPlayerStatesScreenTest_mediaPlayerScreen[1]_notconnected.png:1]: This one seems correct, maybe it was never updated due to the tolerance

File: media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui.components_MediaChipTest_givenNOTitle_thenDisplaysDefaultTitle.png
Comment (luizgrp) [inline media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui.components_MediaChipTest_givenNOTitle_thenDisplaysDefaultTitle.png:1]: This one and above seem correct, maybe also never updated due to tolerance

File: media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui.components_LoadingMediaDisplayTest_loadingMediaDisplay_textMediaDisplay_overlay.png
Comment (luizgrp) [reply media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui.components_LoadingMediaDisplayTest_loadingMediaDisplay_textMediaDisplay_overlay.png:1]: Are you going to raise a bug to track or are going to fix in this one?

File: images/coil/src/test/snapshots/images/com.google.android.horologist.compose.material_ChipTest_withPlaceholderIcon.png
Comment (yschimke) [reply images/coil/src/test/snapshots/images/com.google.android.horologist.compose.material_ChipTest_withPlaceholderIcon.png:1]: https://github.com/google/horologist/issues/1927

File: compose-material/src/test/snapshots/images/com.google.android.horologist.compose.material_ChipTest_withLongText.png
Comment (yschimke) [reply compose-material/src/test/snapshots/images/com.google.android.horologist.compose.material_ChipTest_withLongText.png:1]: Preview shows the same.  
image

Comment (yschimke) [reply compose-material/src/test/snapshots/images/com.google.android.horologist.compose.material_ChipTest_withLongText.png:1]: Similar-ish on device 
image

**Review by luizgrp — APPROVED** [review]

**Pull request merged**